src/helpers/index.js

```diff
@@ -1,3 +1,4 @@
+export * from './bookmarkHelpers';
```

Comment:
filename singular please to align with the others

Reply:
changed it in 8a84e9e25ce56257a4c54ca67a967bcc32fcc310